wire-library/wire-moshi-adapter/src/test/java/squareup/keywords/KeywordKotlin.kt

```diff
@@ -98,12 +114,15 @@ class KeywordKotlin(
     @JvmField
     var fun_: Map<String, String> = emptyMap()
 
+    @JvmField
+    var private: List<Boolean> = emptyList()
```

Comment:
So this is either:
 1. Broken because it's not escaped, or
 2. Not a keyword so shouldn't be used to test this case in Kotlin

Reply:
it compiles so.... will change.

Reply:
![image](https://user-images.githubusercontent.com/1767669/80024483-2ac53f80-84ad-11ea-9e35-3d278f5e03ad.png)

Reply:
used `return` instead for both.